step.yml

```diff
@@ -9,16 +9,17 @@ description: |-
   
   Before you start using this Step, you need to do a couple of things:
   
-  * Register an app on the **My Apps** page of iTunes Connect. Click on the *plus sign* and select the **New App** option. This requires an **admin** account.
+  * Register an app on the **My Apps** page of App Store Connect. Click on the *plus sign* and select the **New App** option. This requires an **admin** account.
   * This Step requires an app signed with App Store Distibution provisioning profile. Make sure that you use the correct code signing files and the correct export method with the Step that builds your binary.
-  * Every build that you want to push to iTunes Connect must have a unique build and version number pair. Increment either or both before a new deploy to iTunes Connect.
+  * Every build that you want to push to App Store Connect must have a unique build and version number pair. Increment either or both before a new deploy to App Store Connect.
```

Comment:
"...to push to the App Store Connect.." and "...both before a new deploy to the App Store Connect".